test/fixtures/api/rippled/account-tx.js

```diff
@@ -196,15 +198,37 @@ module.exports = function(request, options={}) {
     TransactionResult: 'tesSUCCESS'
   };
 
+  let marker = +request.marker || 0;
+  marker += 1;
+  if (marker === 5) {
+    meta.TransactionResult = 'tecINSUFFICIENT_RESERVE';
+  } else if (marker === 6) {
+    tx = _.cloneDeep(AccountSet.result);
+    meta = tx.meta;
+    delete tx.meta;
+  } else if (marker === 7) {
+    tx.Account = addresses.OTHER_ACCOUNT;
+  } else if (marker === 8) {
+    tx.Destination = addresses.THIRD_ACCOUNT;
+  } else if (marker > 25) {
+    marker = undefined;
+  } else if (marker > 15) {
+    tx.Account = addresses.ISSUER;
+    tx.Destination = addresses.ACCOUNT;
+  }
+  if (request.limit === 13) {
+    const res = _.assign({}, NotFound, {id: request.id});
+    return JSON.stringify(res);
+  }
   return JSON.stringify({
     id: request.id,
     status: 'success',
     type: 'response',
     result: {
-      marker: request.marker === undefined ? 'ABC' : undefined,
+      marker: marker === undefined ? undefined : String(marker),
       transactions: [
         {
-          ledger_index: 348860,
+          ledger_index: 348860 - +marker,
```

Comment:
We're going to disallow this soon, so better to change it now: https://github.com/eslint/eslint/issues/1621